### DISCUSSION TITLE
How to remove border box on arc charts when saving to png

### DISCUSSION THREAD

**Discussion by mcp292** [vega/altair#2990] [Q&A] [answered]
# [MWE](https://altair-viz.github.io/gallery/pie_chart.html)
```py
import pandas as pd
import altair as alt


source = pd.DataFrame({"category": [1, 2, 3, 4, 5, 6], "value": [4, 6, 10, 3, 7, 8]})

alt.Chart(source).mark_arc(stroke="black").encode(
    theta=alt.Theta(field="value", type="quantitative"),
    color=alt.Color(field="category", type="nominal"),
).save("arc.png", webdriver="firefox")
```

![arc plot](https://user-images.githubusercontent.com/49123398/227616906-52adc526-cfcf-428b-8f25-5efaa2613530.png)

# Question 
Is there a way to get rid of the border box? It seems to be related to saving to png as the example linked above does not have it. It is present with and without `stroke`.

**Comment by joelostblom** [comment, marked as answer]
This section of the docs shows that https://altair-viz.github.io/user_guide/customization.html#removing-the-chart-border. Actually maybe we should update that to use `stroke=None` which might be more semantacally pleasing (but same effect as `strokeWith=0`.

**Reply by mcp292** [reply]
I'm happy to see this in the docs! Thanks for the link!

Is the default for HTML plots `strokeWidth=0` then?

I think you mean `strokeWidth=None`, in which case I do like that too.

**Reply by joelostblom** [reply]
`stroke=None` seems to work for me. And I see the outline in html as well unless I include that line.

**Reply by mcp292** [reply]
Oh, yes you're correct, my mistake:
```py
alt.Chart(source).mark_arc(stroke="black").encode(
    theta=alt.Theta(field="value", type="quantitative"),
    color=alt.Color(field="category", type="nominal")
).configure_view(
    stroke=None
).save("arc.png", webdriver="firefox")
```
